history.md

```diff
@@ -1,67 +1,75 @@
-# 0.0.1
```

Comment:
Delete this file? 
There is no information of each version. Keeping this fie makes readers confused.

Reply:
make sense.

Reply:
make sense.